packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -0,0 +1,228 @@
+import { Page, FrameLocator } from 'playwright';
+
+export type PreviewOptions = 'Desktop' | 'Mobile' | 'Tablet';
+
+const panel = 'div.interface-interface-skeleton__header';
+const selectors = {
+	// Block Inserter
+	blockInserterButton: `${ panel } button.edit-post-header-toolbar__inserter-toggle`,
+
+	// Draft
+	saveDraftButton: `${ panel } button.editor-post-save-draft[aria-disabled="false"]`,
+	saveDraftDisabledButton: `${ panel } button.editor-post-saved-state.is-saved[aria-disabled="true"]`,
+	switchToDraftButton: `${ panel } button.editor-post-switch-to-draft`,
+
+	// Preview
+	mobilePreviewButton: `${ panel } a.editor-post-preview`,
+	desktopPreviewButton: `${ panel } button.block-editor-post-preview__button-toggle`,
+	desktopPreviewMenuItem: ( target: PreviewOptions ) =>
+		`button[role="menuitem"] span:text("${ target }")`,
+	previewPane: ( target: PreviewOptions ) => `.is-${ target.toLowerCase() }-preview`,
+
+	// Publish
+	publishButton: `${ panel } button.editor-post-publish-button__button[aria-disabled="false"]`,
+
+	// Editor settings
+	settingsButton: `${ panel } .interface-pinned-items > button:first-child`,
```

Comment:
Same question here re: text selectors

Reply:
As with https://github.com/Automattic/wp-calypso/pull/61502#discussion_r818250311, some selectors were changed with UI based ones where it made sense.